package.json

```diff
@@ -23,7 +23,7 @@
     "url": "https://github.com/LambdaSchoolEngineering/postgres-migrations/issues"
   },
   "engines": {
-    "node": ">10.17.0"
+    "node": "14.17.1"
```

Comment:
I updated the node version here. I thought about changing the update-node script to support symbols like `>`, but decided that the script will make it easy enough to update node versions that we'll probably just keep all the packages on the same version of node, so specifying an exact version for a npm package like this should be fine.

If it becomes a pain, then I would recommend modifying the update-node script in third-party-automations to preserve symbols.